plugin/Modules/System.php

```diff
@@ -234,7 +234,7 @@ public function getWordpressDetails()
         global $wpdb;
         $theme = wp_get_theme();
         return [
-            'Active Theme' => sprintf('%s v%s', (string) $theme->Name, (string) $theme->Version),
```

Comment:
The `wp_get_theme()` function returns object keys that are in title case (unfortunately) as it takes the values from the style.css file.

![image](https://user-images.githubusercontent.com/134939/96975213-caf13500-14df-11eb-83ba-89557a3b3dfa.png)

Reply:
The WordPress extension was writted based on

```php
	/**
	 * __isset() magic method for properties formerly returned by current_theme_info()
	 *
	 * @since 3.4.0
	 *
	 * @param string $offset Property to check if set.
	 * @return bool Whether the given property is set.
	 */
	public function __isset( $offset ) {
		static $properties = array(
			'name',
			'title',
			'version',
			'parent_theme',
			'template_dir',
			'stylesheet_dir',
			'template',
			'stylesheet',
			'screenshot',
			'description',
			'author',
			'tags',
			'theme_root',
			'theme_root_uri',
		);

		return in_array( $offset, $properties, true );
	}
```

https://github.com/johnpbloch/wordpress-core/blob/18d3dad02b8c3794df21e430a1adc4e66bf33fc1/wp-includes/class-wp-theme.php#L453-L480

Does these supersede headers?

Reply:
Please also see the following `__get()` as that is tricky.